static/app/views/performance/newTraceDetails/traceHeader/breadcrumbs.tsx

```diff
@@ -375,26 +386,89 @@ function getInsightsModuleBreadcrumbs(
       break;
   }
 
-  crumbs.push({
-    label: t('Trace View'),
-  });
+  crumbs.push(leafBreadcrumb);
 
   return crumbs;
 }
 
-export function getTraceViewBreadcrumbs(
-  organization: Organization,
-  location: Location,
-  moduleUrlBuilder: URLBuilder,
-  view?: DomainView
-): Crumb[] {
+function LeafBreadCrumbLabel({
+  traceSlug,
+  project,
+}: {
+  project: Project | undefined;
+  traceSlug: string;
+}) {
+  const [isHovered, setIsHovered] = useState(false);
+
+  return (
+    <Wrapper
+      onMouseEnter={() => setIsHovered(true)}
+      onMouseLeave={() => setIsHovered(false)}
+    >
```

Comment:
Seems excessive to use a state just to track hover and conditionally render.
I think we can accomplish the same thing here using `:hover` css selector

Reply:
Yup agree! Implemented.